pkgs/top-level/python-packages.nix

```diff
@@ -1178,6 +1178,45 @@ in {
     };
   };
 
+  couchpotato = buildPythonPackage rec {
+    name = "couchpotato-${version}";
+    version = "3.0.1";
+    disabled = isPy3k;
+
+    src = pkgs.fetchurl {
+      url = "https://github.com/CouchPotato/CouchPotatoServer/archive/build/${version}.tar.gz";
+      sha256 = "1xwjis3ijh1rff32mpdsphmsclf0lkpd3phpgxkccrigq1m9r3zh";
+    };
+
+    phases = [ "unpackPhase" "patchPhase" "installPhase" "postInstall" ];
```

Comment:
you might want to pass `format = "other";`. That gives you a `buildPythonPackage` for non-setuptools installations.